internal/build/build.go

```diff
@@ -429,7 +437,62 @@ func buildAllPkgs(ctx *context, initial []*packages.Package, verbose bool) (pkgs
 	return
 }
 
-func linkMainPkg(ctx *context, pkg *packages.Package, pkgs []*aPackage, conf *Config, mode Mode, verbose bool) {
+var (
+	globalNames = make(map[string][]string)
```

Comment:
Is it possible not to create global variables?

Reply:
move GlobaNames to build.Config